LICENSE-HEADER

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
Why do we need to have it? We need to remove it from PS operator too.

Reply:
controller-gen puts this to the beginning of auto generated files

Reply:
please do not commit it because we will have
<img width="362" alt="Screen Shot 2022-08-24 at 11 30 26" src="https://user-images.githubusercontent.com/4948913/186370455-95312633-d013-4614-9098-beca2cf0c6cb.png">